api/src/Repository/UserRepository.php

```diff
@@ -341,4 +341,20 @@ public function deleteInactiveAdminUsers(array $inactiveAdminUserIds)
 
         return $result->getResult();
     }
+
+    public function deleteStaffUsersThatAreNotInSirius(array $staffUsersToBeDeleted)
+    {
+        $em = $this->getEntityManager();
+        $rsm = new ResultSetMappingBuilder($em);
+
+        $sql = 'DELETE FROM dd_user WHERE id IN (:ids)';
+        $params = [
+            'ids' => $staffUsersToBeDeleted,
+        ];
+
+        $stmt = $em->createNativeQuery($sql, $rsm);
```

Comment:
Have you checked this works on Dev? I was under the impression createNativeQuery() was just for SELECT queries rather than updates/deletes...  (if it works it works, though!) https://www.doctrine-project.org/projects/doctrine-orm/en/2.16/reference/native-sql.html#the-nativequery-class